internal/api/external.go

```diff
@@ -390,7 +390,8 @@ func (a *API) createAccountFromExternalIdentity(tx *storage.Connection, r *http.
 		if !emailData.Verified && !config.Mailer.Autoconfirm {
 			mailer := a.Mailer(ctx)
 			referrer := a.getReferrer(r)
-			if terr = sendConfirmation(tx, user, mailer, config.SMTP.MaxFrequency, referrer, config.Mailer.OtpLength); terr != nil {
+			// TODO (joel) - add PKCE
+			if terr = sendConfirmation(tx, user, mailer, config.SMTP.MaxFrequency, referrer, config.Mailer.OtpLength, models.ImplicitFlow); terr != nil {
```

Comment:
This todo supposed to be here?

Reply:
Yup, wanted to highlight the reason why it's using implicit flow and not making use of `flowType`

Reply:
maybe we should just remove this block altogether:

```
if !emailData.Verified && !config.Mailer.Autoconfirm {
  ...
}
```

This check is basically for cases where the oauth email is not verified, then we send an email confirmation to the user. But maybe we should simply return an error here and ask the user to confirm their email with the oauth provider?

Reply:
Removed the TODO for now, will open another PR for removing the check as it involves editing some tests